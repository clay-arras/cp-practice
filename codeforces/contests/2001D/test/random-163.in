1
24
23 17 16 5 25 4 19 20 8 1 19 2 23 12 25 20 10 9 20 5 13 18 10 24
