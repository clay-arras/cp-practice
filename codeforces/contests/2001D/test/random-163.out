17
23 17 16 4 20 8 1 19 2 12 25 9 5 13 18 10 24
